src/tasks/spanish/teleia_siele.py

```
@@ -0,0 +1,81 @@
+    
+    strict_parsing = False
+    labels = {0: "A", 1: "B", 2: "C"}
+    
+    user_prompt_template = "{text}\n\nOpciones:\n{choices}\n\nRespuesta:"
+    
+    def _download_and_cache(self, output_path: Path):
+        """Transform Teleia SIELE dataset to eval format."""
+        try:
+            from datasets import load_dataset
+        except ImportError:
+            raise ImportError("datasets library required. Install with: pip install datasets")
```

Ruff (0.14.10)

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)